Changelog

```diff
@@ -1,19 +1,24 @@
 Version 3.3, *2020-XX-XX*
 
   Features:
-    * New token type: WebAuthn/FIDO2 token (#1468)
-      WebAuthn hardware tokens can now be used with privacyIDEA
+    * New token type: WebAuthn/FIDO2 token is initially supported by privacyIDEA (#1468)
     * New token type: Indexed Secret token allows user
       to authenticate with a pre-known secret that can be
       initialized from the user store. (#1986)
+    * New Event Handler Module: Logging module enables custom event-driven logging (#1580)
 
   Enhancements:
-    * Added the remote serial to the tokeninfo of a remote token to better track
-      authenticated devices (#2031)
+    * API
```

Comment:
Everything is an API. We have not used this before. Please revert this.

Reply:
moved to Misc